package.json

```diff
@@ -19,43 +19,34 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w ./lib",
+    "lint": "jshint lib/**/*.js test/**/*.js --config .jshintrc",
```

Comment:
Please change as we discussed for the IOTA-JSON agent:

```
"lint": "jshint lib/ --config .jshintrc && jshint test/ --config test/.jshintrc",
```

Reply:
Fixed with b522e66af5b7c0d039c440470c4332d97521ef83